src/modules/mavlink/mavlink_main.h

```diff
@@ -123,11 +123,9 @@ class Mavlink : public device::VDev
 
 	static void		forward_message(const mavlink_message_t *msg, Mavlink *self);
 
-#ifndef __PX4_QURT
 	static int		get_uart_fd(unsigned index);
```

Comment:
@mcharleb Does QURT support network or serial or none? The IFDEF seems odd, if none is supported, shouldn't it just stop building this app overall?